### PR TITLE
CMR-7028: Refactored search app imports to solve excess warning messages

### PR DESCRIPTION
**Pull request by mitchstartzel**

### CODE REVIEW

**Review by jaybarra — APPROVED** [review]

File: search-app/project.clj [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
   :url "https://github.com/nasa/Common-Metadata-Repository/tree/master/search-app"
-  :exclusions [[cheshire]
-               [clj-time]
-               [com.fasterxml.jackson.core/jackson-core]
-               [commons-codec/commons-codec]
-               [org.apache.httpcomponents/httpclient]
-               [org.clojure/clojure]
-               [org.clojure/tools.reader]
-               [ring/ring-codec]]
+  :exclusions [[org.clojure/clojure]
```
Comment (ygliuvt) [inline search-app/project.clj:4]: sort the exclusions libraries in alphabetic order.

File: search-app/project.clj [hunk 2]
```diff
@@ -1,17 +1,21 @@
...
+               clj-time
+               org.eclipse.emf/org.eclipse.emf.common
+               org.apache.httpcomponents/httpclient
+               org.clojure/tools.reader
+               org.eclipse.emf/org.eclipse.emf.ecore
+               com.fasterxml.jackson.core/jackson-core]
   :dependencies [[cheshire "5.8.1"]
                  [clj-time "0.15.1"]
                  [com.fasterxml.jackson.core/jackson-core "2.12.0"]
+                 [com.fasterxml.jackson.core/jackson-annotations "2.12.0"]
```
Comment (ygliuvt) [inline search-app/project.clj:17]: sort in alphabetic order.

File: search-app/project.clj (hunk 1 above)
Comment (mitchstartzel) [reply search-app/project.clj]: Fixed.

File: search-app/project.clj (hunk 2 above)
Comment (mitchstartzel) [reply search-app/project.clj:16]: Fixed.

**Review by jceaser — APPROVED** [review]

**Review by ygliuvt — APPROVED** [review]

**Review by yiduoke — APPROVED** [review]

**Pull request merged**